docs/UserGuide.adoc

```diff
@@ -328,6 +328,44 @@ There is no need to synchronize manually.
 Stops synchronizing of contact information with any previously configured cloud accounts. +
 Format: `unsync`
 
+== Facebook Integration [Coming in v2.0]:
```

Comment:
Was thinking if we should (again) consolidate this into a single command with options?
e.g.
`facebook`
`facebook connect`
`facebook addall`
`facebook addcontact`